### PR TITLE
Refactored code to fix a bug

### PR DESCRIPTION
**Pull request by CoriBeecroft**
Fixed bug.

### CODE REVIEW

File: Node.js [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+	  }
+	  else if(!inChild && i>0)
+	  {
+	  	nodeText += character;
+
+	  }
+	}
+	this.content.push(nodeText.slice(contentPieceStart));
+}
+
```
Comment (westoncb) [inline Node.js]: Nice name!

File: Node.js [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+Node.prototype.determineTypeAndRemoveHeader = function(string)	//Cori, this is kind of hard to understand, might want to refactor or add some more comments. 
+{
```
Comment (westoncb) [inline Node.js]: Nice name!

File: Node.js [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+	  else if(!inChild && i>0)
+	  {
+	  	nodeText += character;
+
+	  }
+	}
+	this.content.push(nodeText.slice(contentPieceStart));
+}
+
+Node.prototype.determineTypeAndRemoveHeader = function(string)	//Cori, this is kind of hard to understand, might want to refactor or add some more comments.
```
Comment (westoncb) [inline Node.js]: Nice name!

**Comment by westoncb** [comment]
Looks good! Go ahead and merge! 👍

**Comment by CoriBeecroft** [comment]
=D

**Pull request merged**